Added Frank Schmitt to Authors

frankschmitt opened the pull request:
@dankelley Sure. I put my name in the Author: List in the DESCRIPTION file - feel free to edit it if necessary, and thanks for the quick response to #20 !

dankelley commented:
Thanks!

dankelley commented:
Actually, 
1. I think it should be called `as.burndown()` which is the usual R convention
2. can you move the code into `R/burndown.R` please?

I want you to be the one doing these, so that git will know that you are the one doing them, i.e. so you'll get the credit.

frankschmitt commented:
@dankelley  I've renamed it to as.burndown() and moved the code to burndown.R. One question before I create the PR: checking the package seems to have changed all the files in the man/ subdirectory and deleted two files in inst/ (possibly because I'm running a different roxygen version).
Should I include these changes in the PR? Or just the changes in the R code / NAMESPACE ?

dankelley commented:
I think you ought to update just the R and NAMESPACE.
